pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -119,6 +129,8 @@ stdenv.mkDerivation rec {
     install_name_tool -change $out/lib/libinkscape_base.dylib $out/lib/inkscape/libinkscape_base.dylib $out/bin/inkview
   '';
 
+  ninjaFlags = [ "-v" ]; # make ninja verbose
```

Comment:
Why this?

Reply:
Don't we want the logs in the CI? If not, I'll remove this.

Reply:
I think the default verbosity should be sufficient. If we want this, we should consider doing it globally in ninja setup hook.

Reply:
Addressed in  863becb